fibonacci-series.php

```diff
@@ -0,0 +1,26 @@
+<?php 
+// PHP program to print first n Fibonacci numbers 
+  
+// Function to print first n Fibonacci Numbers 
+
+function printFibonacciNumbers($n) 
+{ 
+    $f1 = 0; 
```

Comment:
Codacy found an issue: [Avoid variables with short names like $f1. Configured minimum length is 3.](https://app.codacy.com/gh/fharookshaik/fibonacci-series/pullRequest?prid=6328068)